crates/matrix-sdk/src/event_cache/pagination.rs

```diff
@@ -185,7 +185,7 @@ impl RoomPagination {
                 let first_event_pos = room_events.events().next().map(|(item_pos, _)| item_pos);
 
                 // First, insert events.
-                let insert_new_gap_pos = if let Some(gap_id) = prev_gap_id {
```

Comment:
Are you doing something with `add_event_report`?

Reply:
(Yep, it's used in a subsequent commit :+1:)